removeFromArray/removeFromArray.js

```diff
@@ -2,4 +2,31 @@ const removeFromArray = function() {
 
 }
 
+/* Try without using forEach()
+
+const removeFromArray = function(array, ...toRemove) {
+    let testArray = [the array to be passed in];
+    let newArray = [];
+    for (let i = 0; i < array.length; i++) {
+        if (testArray[i] !== toRemove) {
```

Comment:
`toRemove` is an array-like container of arguments and it will never be equal to an individual element of the test array. You need to be looking at the individual elements of `toRemove`. When you need to look at each value of an array, that means you need to iterate. That means you need a loop! So you need a `for` loop inside a `for` loop. Your first (or "outer" loop) will iterate over the test array just as you've done. But immediately inside this loop, you need to start a second ("inner") loop to iterate over the `toRemove` array.

With nested loops, you must use a different counter variable name than in the outer loop. This is due to variable scope.

Bad:

```
for (i=0; i < array.length; i++) {
    for (i=0; i < toRemove.length;  i++) {
        ...
    }
}
```

The convention is to use `i` for the first loop, `j` for the next one, then `k`, and so on.